analysis/topEFT/topeft.py

```diff
@@ -400,10 +402,23 @@ def process(self, events):
           selections.add("is_good_lumi",lumi_mask)
 
           # 2lss selection
-          selections.add("2lss_p", (events.is2l & chargel0_p & bmask_atleast1med_atleast2loose & pass_trg)) # Note: The ss requirement has NOT yet been made at this point! We take care of it later with the appl axis
-          selections.add("2lss_m", (events.is2l & chargel0_m & bmask_atleast1med_atleast2loose & pass_trg)) # Note: The ss requirement has NOT yet been made at this point! We take care of it later with the appl axis
+          # Drained of 4 top
+          selections.add("2lss_p", (
+                    events.is2l & chargel0_p & bmask_atleast1med_atleast2loose & pass_trg & bmask_atmost3med))  # Note: The ss requirement has NOT yet been made at this point! We take care of it later with the appl axis
+          selections.add("2lss_m", (
+                    events.is2l & chargel0_m & bmask_atleast1med_atleast2loose & pass_trg & bmask_atmost3med))  # Note:
+          # The ss requirement has NOT yet been made at this point! We take care of it later with the appl axis
+		
+        # Enriched in 4 top
+          selections.add("2lss_4t_p", (
+                    events.is2l & chargel0_p & bmask_atleast1med_atleast2loose & pass_trg & bmask_atleast3med))  # Note: The ss requirement has NOT yet been made at this point!
+          # We take care of it later with the appl axis
+          selections.add("2lss_4t_m", (
+                  events.is2l & chargel0_m & bmask_atleast1med_atleast2loose & pass_trg & bmask_atleast3med))  # Note:
+          # The ss requirement has NOT yet been made at this point! We take care of it later with the appl axis
+		
```

Comment:
Similar to the discussion in the other comment, we might want to keep these on one line as well (for consistency with the rest of the formatting).

Reply:
I will apply this in the next version of the code